package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "typesctipt-for-nodejs",
+  "version": "1.0.0",
+  "description": "## Introduction",
+  "main": "index.js",
+  "scripts": {
+    "build": "npx tsc",
+    "build:watch": "npx tsc --watch",
+    "start": "node dist/index.js",
+    "start:watch": "nodemon dist/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fragmnt02/typesctipt-for-nodejs.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/fragmnt02/typesctipt-for-nodejs/issues"
+  },
+  "homepage": "https://github.com/fragmnt02/typesctipt-for-nodejs#readme",
+  "devDependencies": {
+    "@types/express": "^4.17.9",
+    "dotenv": "^8.2.0",
+    "nodemon": "^2.0.6",
+    "typescript": "^4.1.2"
+  },
```

Comment:
I like how you've already gone ahead with the work on your routers but there's a detail here:
dotenv should be a normal dependency because if you package it as a devDependency your production app won't be able to read .env files.